exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -171,10 +281,67 @@ impl EjbCommand for Run {
     }
 }
 
+/// There are four possible combinations of parameters:
+/// 1. Neither `<db_path>`, nor `<node_config>` are provided. `<tmp>` directory is used for database
+///   files and <tmp>/config for node configuration files.
+/// 2. Only `<db_path>` is provided. `<db_path>` is used for database files, `<db_path>/config` is
+///   used for configuration files. If `<db_path>/config` exists, existing configuration files are
+///   used, otherwise node configuration is auto-generated.
```

Comment:
Auto-generated or extracted from the database? If the first, would it even work?